src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -158,16 +180,14 @@ void GeantPhysicsList::add_gamma_processes()
  * - Coulomb scattering and multiple scattering (high E) are currently
  *   disabled.
  */
-void GeantPhysicsList::add_e_processes()
+void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
 {
-    auto       physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
-    const auto electron     = G4Electron::Electron();
-    const auto positron     = G4Positron::Positron();
+    auto* physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
 
-    if (true)
```

Comment:
I noticed you removed the `if (true)` from this bit, which made me think about all these `if (true)` statements that have been here for awhile as placeholders...
Do we want to keep them up for this PR, or should we expand the `GeantPhysicsOptions` to include the processes already?

Reply:
Ah good point: I could make it `if (p == G4Positron::Positron() && true)`? I'd like to basically merge the "geant4 benchmark" physics setup routines into our GeantSetup, so all of the physics here could become on/off.

Reply:
Exactly. I thought the same, and since we can either use the exported root file from benchmarks or use a manually selected set of options for the `celer-export-geant`, at the end we should have equivalent input options. In the first case, we just check the loaded processes/models, but for the second we will end up with a list of processes to turn on/off. So for now the `if (p == G4Positron::Positron() && true)` is fine, but in the end I think we will end up with something like:

```c++
if (p == G4Positron::Positron() && options_.annihilation)
```

Same goes for the Klein Nishina Compton process, where the `if (true)` was removed.